doc/releases/changelog-0.36.0.md

```diff
@@ -215,23 +215,45 @@
 
 <h4>Simulate mixed-state qutrit systems 3️⃣</h4>
```

Comment:
```suggestion
<h3>Improvements 🛠</h3>

<h4>Simulate mixed-state qutrit systems</h4>
```
Let's make this part of "improvements" since there are follow up part of this coming in a future release.